### PR TITLE
E-mail attribute filtered explanation

### PR DESCRIPTION
**Pull request by MiguelAHM**
- Explain why the email is filtered in contact entities
- Explain how to do to get the email of those contact entities
- Explain why notify and rest of notification attributes are not contemplated in the output

### CODE REVIEW

File: README.RDAP.md [hunk 1]
```diff
@@ -45,21 +45,22 @@ For example: https://rdap.db.ripe.net/entity/KR4422-RIPE
 Related Contact information is Filtered
 ---------------------------------------
 All contact entities ("technical","administrative" etc.) have filtered contact information, i.e. "e-mail"
```
Comment (eshryane) [inline README.RDAP.md:47]: Add the word "related" to the first sentence, i.e. "All related entities... have filtered contact information..."

File: README.RDAP.md [hunk 2]
```diff
@@ -45,21 +45,22 @@ For example: https://rdap.db.ripe.net/entity/KR4422-RIPE
...
-and "notify" values are not included. Contact entities do not count towards the daily query limit
+value is not included. Contact entities do not count towards the daily query limit
```
Comment (eshryane) [inline README.RDAP.md:48]: Clarify that *no* e-mail is included, "i.e. no e-mail addresses are included." (to include *all* attributes containing an email address, don't specify which attributes as there are quite a few of them)

Also replace the term "Contact entities" with "Related entities" to be consistent.

File: README.RDAP.md [hunk 3]
```diff
@@ -45,21 +45,22 @@ For example: https://rdap.db.ripe.net/entity/KR4422-RIPE
...
 https://www.ripe.net/manage-ips-and-asns/db/support/documentation/ripe-database-acceptable-use-policy.
 This was done to avoid blocking clients for inadvertently querying excessively for personal data.
 
-For entity request "e-mail" will be enabled and the clients requesting those service will need to comply with the daily limit
+For entity requests "e-mail" is enabled and the clients requesting this service need to comply with the daily limit
```
Comment (eshryane) [inline README.RDAP.md:52]: Re-word, "For entity responses, the contact information is not filtered, i.e. the "e-mail" attribute is included.".

(Mirror the sentence above)

Comment (eshryane) [inline README.RDAP.md:52]: Separate sentence, "Clients making entity requests must comply with the daily limit specified in the Acceptable Use Policy" (and make the "Acceptable Use Policy" a link).

As in, [Acceptable Use Policy](https://www.ripe.net/manage-ips-and-asns/db/support/documentation/ripe-database-acceptable-use-policy)

File: README.RDAP.md [hunk 4]
```diff
@@ -45,21 +45,22 @@ For example: https://rdap.db.ripe.net/entity/KR4422-RIPE
...
 according to the Acceptable User Policy (AUP): https://www.ripe.net/manage-ips-and-asns/db/support/documentation/ripe-database-acceptable-use-policy.
+So this is the way to get the "e-mail" of the contact entities.
```
Comment (eshryane) [inline README.RDAP.md:54]: Remove this sentence, I think it's redundant (i.e. it should be clear from the text further up when email is returned and when not).

File: README.RDAP.md [hunk 5]
```diff
@@ -45,21 +45,22 @@ For example: https://rdap.db.ripe.net/entity/KR4422-RIPE
...
 
-For the rest of request either "e-mail" is filtered. Abuse contact is always returned, and
+For the rest of the requests either "e-mail" is filtered. Abuse contact is always returned, and
```
Comment (eshryane) [inline README.RDAP.md:56]: "For non-entity requests, e-mail addresses are filtered, except for the abuse contact which is always returned."

Separate sentence for the other attribute types.

File: README.RDAP.md (hunk 2 above)
Comment (MiguelAHM) [reply README.RDAP.md:48]: This is already explained in the last paragraph. I think that it is redundant to talk about the request that have e-mail filtered.
I am ok with the rest of the comments.

**Review by eshryane — APPROVED** [review]

**Pull request merged**